### PR TITLE
Add psion battlerage sorting

### PR DESCRIPTION
**Pull request by tsartin**

### CODE REVIEW

File: script.lua [hunk 1]
```diff
@@ -175,6 +175,19 @@ local sortDepthswalkerBattlerage = function()
...
 end
 
+local sortPsionBattlerage = function()
+
+	debugMessage("sorting brage for psion", {battlerageSkills = battlerageSkills})
+
+	if class ~= "Psion" or #battlerageSkills ~= 6 then return end
+	battlerageSkills[2], battlerageSkills[3], battlerageSkills[4] = battlerageSkills["regrowth"], battlerageSkills["pulverise"], battlerageSkills["devastate"]
+
+	battlerageSkills["terror"].affliction = "fear"
```
Comment (keneanung) [inline script.lua:185]: Is this the only battlerage ability that is missing the affliction? Does the "combo" ability have the used afflictions set?

**Review by keneanung — COMMENTED** [review]
Hey there. Thank you for your contribution! I have a question to ask before I merge this.

**Review by keneanung — APPROVED** [review]

**Comment by tsartin** [comment]
Apologies for the late reply. been a little out of pocket.

I don't have psion myself. I'm thinking that terror was the only one that was missing an affliction and the other person didn't think the combo ability used an affliction. This is the information they shared though.

"Barbedblade does DoT damage."
"Devastate does damage."
"Regrowth calls forth binding vines to grasp your foe, preventing them from being able to heal. no aff or requirements lists."
"Pulverise breaks shield"
"Whirlwind says causes massive damage. no aff or anything required."
"Then terror gives fear"

**Comment by keneanung** [comment]
All right. I was thinking I'd merge this as it's an improvement and then fix things on the go.

**Pull request merged**